### PR TITLE
Fix Storybook form input props

### PR DESCRIPTION
**Pull request by joseabernardes**
## Summary

- [Ticket](#)
- [Design](#)
- [Storybook](#)

### General

- [ ] [Changeset added](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md)
- [ ] The submitted code is organized and formatted according to our [💅 Style Guide](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-style-guide--docs).
  - [ ] The code is formatted with Prettier.
  - [ ] There are no linting errors.

### For new or updated components

https://swirl-storybook.flip-app.dev/?path=/docs/contributions-merge-publish--page

- [x] The changes do not contain any breaking changes.
- [x] The changes do not introduce new components that don't belong in the library (e.g. non-reusable components, highly specialized components, components including business logic)
- [ ] The component documentation is updated.
- [ ] The changes meet the [🤖 testing requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-testing--docs).
  - [ ] New features are tested.
  - [ ] In case of bug fixes, regression tests have been added.
  - [ ] All tests are 🟢.
- [ ] The changes meet the [🧏‍♀️ accessibility requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-accessibility--docs).
  - [ ] WCAG 2.1 Level A and Level AA requirements are met.
  - [ ] The Storybook a11y addon shows no errors.
  - [ ] The changes have been tested with a screen reader.
  - [ ] Keyboard controls have been tested, if applicable.
  - [ ] Components implementing form controls (inputs, buttons, selects, etc.) do not use Shadow DOM, and instead use Stencil's scoping mechanism
- [ ] The changes use our [🌈 theming concept](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-theming--docs).
  - [ ] Design tokens have been used where appropriate.
  - [ ] The component has been visually checked in combination with the "Light" and "Dark" theme.
- [ ] The changes meet our [🌍 internationalization requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-internationalization--docs).
  - [ ] No static text is used.
  - [ ] The component doesn't break with longer texts or different text wrappings.
  - [ ] Number, currency and date values can be formatted appropriately.
- [ ] The changes work in all supported browsers and viewports. See [📱 Responsive Design](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-responsive-design--docs)

### CODE REVIEW

**Comment by joseabernardes** [comment]
Does it make sense to add a new changelog entry in this case? I'm asking because the PR only changes storybook files, and not actual lib code.

File: packages/swirl-components/src/components/swirl-color-input/swirl-color-input.stories.ts [hunk 1]
```diff
@@ -16,8 +16,7 @@ const Template = (args) => {
   const formControl = document.createElement("swirl-form-control");
   const element = generateStoryElement("swirl-color-input", args);
 
-  formControl.setAttribute("label", "Color input");
-  formControl.setAttribute("disabled", args.disabled ? "true" : "false");
+  addArgumentsToElement(formControl, { ...args, label: "Color input" });
```
Comment (joseabernardes) [inline packages/swirl-components/src/components/swirl-color-input/swirl-color-input.stories.ts:19]: I was a bit undecided about the approach to take here...

Initially I thought about creating a helper function in which I specifically added the attributes (description, disabled, inline, invalid, label ), but then I thought that if in the future we add any more attributes to the form-control, we would have to remember to update the function.

So I thought of this solution of injecting all the attributes of the input into the form-control. The disadvantage of this solution is that we'll be adding attributes we don't need to the form-control.
In your opinion, does that feel unsafe? Should I go with the more declarative approach?

**Comment by Sqrrl** [comment]
> Does it make sense to add a new changelog entry in this case? I'm asking because the PR only changes storybook files, and not actual lib code.

No. No changeset necessary.

File: packages/swirl-components/src/components/swirl-color-input/swirl-color-input.stories.ts (hunk 1 above)
Comment (Sqrrl) [reply packages/swirl-components/src/components/swirl-color-input/swirl-color-input.stories.ts:19]: `args` only contains custom props that should always be distinct from native HTML attributes. So adding them to components that might not support them should not be an issue.

Comment (Sqrrl) [inline packages/swirl-components/src/components/swirl-color-input/swirl-color-input.stories.ts:19]: Suggestion: `addArgumentsToElement`-> `addAttribtuesToElement`

Comment (joseabernardes) [reply packages/swirl-components/src/components/swirl-color-input/swirl-color-input.stories.ts]: Sounds good!

**Review by Sqrrl — APPROVED** [review]

**Pull request merged**